traits/Sluggable.php

```diff
@@ -0,0 +1,113 @@
+<?php namespace RainLab\Translate\Traits;
+
+use October\Rain\Support\Str;
+
+trait Sluggable
+{
+    /**
+     * Adds translatable slug attributes to the translation data
+     * @return void
+     */
+    protected function slugAttributesTranslated($locale)
+    {
+        if (!isset($this->model->slugs) || !method_exists($this->model, 'slugAttributes')) return;
```

Comment:
@mariavilaro Please add braces around any condition blocks, as per [PSR-2](https://www.php-fig.org/psr/psr-2/) guidelines.
```suggestion
        if (!isset($this->model->slugs) || !method_exists($this->model, 'slugAttributes')) {
            return;
        }
```